### PR TITLE
Add sending locale to backend; fix login localization

### PR DESCRIPTION
**Pull request by voonya**

### CODE REVIEW

File: packages/shared/src/helpers/https.ts [hunk 1]
```diff
@@ -25,11 +25,11 @@ interface MakeRequest {
...
   private _baseUrl: string;
 
-  private _auth: IAuthHelper | null;
+  private _storageService: IStorageService | null;
 
-  constructor(baseUrl: string, auth?: IAuthHelper) {
+  constructor(baseUrl: string, storageService?: IStorageService) {
     this._baseUrl = baseUrl;
-    this._auth = auth;
+    this._storageService = storageService || null;
```
Comment (dmitriy-uvin) [inline packages/shared/src/helpers/https.ts:32]: Storage and Auth are two different entities, it's not correct to tight it in one God like entity, please add third service as Storage

File: packages/shared/src/helpers/https.ts [hunk 2]
```diff
@@ -101,10 +101,13 @@ class Http {
 
     if (acceptLanguage) {
       headers[HttpHeader.ACCEPT_LANGUAGE] = acceptLanguage;
+    } else {
```
Comment (dmitriy-uvin) [inline packages/shared/src/helpers/https.ts:104]: If we have value for locale in our storage, then only then we add that header

File: packages/frontend/pages/_app.tsx [hunk 3]
```diff
@@ -22,6 +23,8 @@ type AppPropsWithLayout = AppProps & {
 const App = ({ Component, pageProps }: AppPropsWithLayout) => {
   const getLayout = Component.getLayout ?? ((page) => page);
 
+  locale.setLocale(i18n.language);
```
Comment (dmitriy-uvin) [inline packages/frontend/pages/_app.tsx:26]: I thin we need update locale value using user's date in other case backend will use default locale

Comment (dmitriy-uvin) [reply packages/frontend/pages/_app.tsx]: How we will change it if user has another locale?

Comment (Nightlngale) [reply packages/frontend/pages/_app.tsx]: default locale will detect i18n

Comment (dmitriy-uvin) [reply packages/frontend/pages/_app.tsx]: ok

**Review by Nightlngale — APPROVED** [review]

**Pull request merged**